### DISCUSSION TITLE
Keyball 44のキーが１つだけ反応しない、ダイオードのランド剥がれ

### DISCUSSION THREAD

**Discussion by shilver-sky** [Yowkees/keyball#498] [Q&A]
Keyball44 の　右手トラックボール側の青丸のキーのみ反応しません。
心当たりとして赤丸の位置のダイオードのランドがはがれています。
エナメル線やゴム被覆線等でリカバリーが可能であればご教示ください。
お忙しいところ恐縮ですがよろしくお願いいたします。
![keyball44](https://github.com/Yowkees/keyball/assets/162725384/615c8c92-db23-4abd-906d-d6f9b778b3cc)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

反応ないキーに対応するダイオードがランドが剥がれているダイオードになりますので、そちらが原因で間違いないかと思われます。
添付しました写真の赤い点とランドが剥がれている部分をエナメル線かゴム被覆線でつないでみてください。
![311290537-615c8c92-db23-4abd-906d-d6f9b778b3cc](https://github.com/Yowkees/keyball/assets/143698349/1ba54a9c-fb76-4814-8559-ade4a35c47f0)

**Reply by shilver-sky** [reply]
ご回答いただきありがとうございます。
指示いただいた修正で無事認識するようになりました。
助かりました！

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます！
ぜひ末永くご愛用いただければ幸いです。
